pkg/webhooks/resource/handlers.go

```diff
@@ -72,6 +73,7 @@ func NewHandlers(
 	eventGen event.Interface,
 	openApiManager openapi.ValidateInterface,
 	admissionReports bool,
+	policyExceptions pcex.Exceptions,
```

Comment:
Since the package is already called `policyexceptions` you can simply name the interface as `Interface`. 

Also, the variable could be shortened to `exceptions`.

Reply:
Sounds good! Thx for letting me know about the good convention 😄